src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -46,7 +49,8 @@ public void resetData_withValidReadOnlyAddressBook_replacesData() {
     @Test
     public void resetData_withDuplicatePersons_throwsDuplicatePersonException() {
         // Two persons with the same identity fields
-        Person editedAlice = new PersonBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND)
+        Person editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB)
+                .withAddress(VALID_ADDRESS_BOB).withUniversity(VALID_UNIVERSITY_BOB).withTags(VALID_TAG_HUSBAND)
```

Comment:
Same as line 80 - 81

Reply:
You shd only change the Address, University, and Tags as they are the non identity fields